config/locales/en.yml

```diff
@@ -436,11 +436,11 @@ en:
           - "Yes"
           - "Not sure"
         items:
-          - text: "What to do if you have lost your job"
+          - text: "What to do if you've lost your job"
             href: "https://www.gov.uk/guidance/coronavirus-covid-19-what-to-do-if-you-were-employed-and-have-lost-your-job"
           - text: "What to do if you’re laid-off"
             href: "https://www.gov.uk/lay-offs-short-timeworking"
-          - text: "Find out about your rights if you have been dismissed"
+          - text: "Find out about your rights if you've been dismissed"
```

Comment:
```suggestion
          - text: "Find out about your rights if you‘ve been dismissed"
```

Reply:
yep, should be the type of apostrophes we use, wasn't intentional!